packages/terra-clinical-application/package.json

```diff
@@ -33,7 +33,6 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-base": "^3.0.0",
```

Comment:
Do we want to bump the peerDependency to the latest terra-base version? Should lerna be doing that automatically?

Reply:
It appears lerna hasn't been bumping peer dependencies 😞

Reply:
I wonder if it will only bump the peer dependency if they are in the same monorepo

Reply:
I believe lerna does not handle bumping peerDeps. We have to do that manually. I think leaving this at `^3.0.0` is fine or setting it to `^3.12.0` is also fine. Both pull in the latest minor version of v3, its just a diff in what the lower bound version be. As long as the v3 minor and patch releases are following semver, using `^3.0.0` for the peerDep shouldn't be an issue.